depthai_nodes/ml/parsers/__init__.py

```diff
@@ -12,7 +12,6 @@
 from .mlsd import MLSDParser
 from .parser_generator import ParserGenerator
 from .ppdet import PPTextDetectionParser
-from .classification_sequence import ClassificationSequenceParser
```

Comment:
Why removing this?

Reply:
`from .classification_sequence import ClassificationSequenceParser` is present in line 3. I think this is a consequence of rebranching.

Reply:
Ah okay, I missed that.